app/screens/browse_channels/channel_dropdown.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {View, Text} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import {useTheme} from '@context/theme';
+import {
+    makeStyleSheetFromTheme,
+} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import {showModalOverCurrentContext} from '../navigation';
+
+import {PUBLIC, SHARED} from './constants';
+import DropdownSlideup from './dropdown_slideup';
+
+type Props = {
+    typeOfChannels: string;
+    onPress: (channelType: string) => void;
+    canShowArchivedChannels: boolean;
+    sharedChannelsEnabled: boolean;
+}
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        channelDropdown: {
+            ...typography('Body', 100, 'SemiBold'),
```

Comment:
On android emulator, this is not showing the text as bold.

Reply:
@shazm any ideas why it may be happening as @larkox is describing?